ros2topic/ros2topic/verb/bw.py

```diff
@@ -58,6 +58,18 @@ def positive_int(string):
     return value
 
 
+def str_bytes(num_bytes):
+    return f'{num_bytes:.0f} B'
+
+
+def str_kilobytes(num_bytes):
+    return f'{num_bytes/1000:.2f} KB'
```

Comment:
Shouldn't this be divided by 1024?

Same below.

Reply:
depends on one's opinion about kibi vs kilo

https://en.wikipedia.org/wiki/Kibibyte
https://en.wikipedia.org/wiki/Kilobyte

Reply:
Internet searches suggest it's debatable :man_shrugging: 

If we divide by 1024, then we should probably also change the units to `KiB`.

Reply:
+1 to `KiB`